code/parse/sexp.cpp

```diff
@@ -19389,7 +19262,10 @@ void unset_nav_carry_status(int node)
 			{
 				if (Ships[i].objnum != -1 && !stricmp(Ships[i].ship_name, name))
 				{
-                    Ships[i].flags.remove(Ship::Ship_Flags::Navpoint_carry);
+					if (set_it)
+	                    Ships[i].flags.set(Ship::Ship_Flags::Navpoint_carry);
+					else
+						Ships[i].flags.remove(Ship::Ship_Flags::Navpoint_carry);
```

Comment:
This could be simplified to include the boolean in the `set()` call: `Ships[i].flags.set(Ship::Ship_Flags::Navpoint_carry, set_it)`

Reply:
Aha.  Thanks.